### ISSUE TITLE
Throw... Anything Causes Insta Crash

### ISSUE THREAD

**Issue by Hunteil** [miki151/keeperrl#752]
Using Alpha 17 ASCII, Windows 7, 64
Use a unit (monster/creature/adventurer) and pick something to throw. i.e. potion of <insert name>. Causes the game to crash almost instantaneously.

This same event occured while casting a spell with a (orc shaman), once it asked for a target, it crashed immediately. Leading me to believe maybe the targeting procedure is failing. But I'm not sure.

I couldn't find an easier way to give debug data to you (I'm not the most skilled with working with hard code). If there's something built in like you did in April with the debug version of the game. I could run that and give you the file, but I didn't find anything in the files called debug.

**Comment by miki151** [comment]
Hey,

Please send me a file called stacktrace.out, if there is one. (or paste it
here)

Thanks.

**Comment by Hunteil** [comment]
I just tried to chat between 2 peasants and it crashed. I was able to read that it was asking for which target I wanted to chat between. Here's the file text you asked for:

FATAL renderer.cpp:444 : tileCoords.count(name) is false. Tile not found arrow0
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
'addr2line' is not recognized as an internal or external command,
operable program or batch file.
